test/e2e/run_ns_test.go

```diff
@@ -0,0 +1,68 @@
+package integration
+
+import (
+	"os"
+
+	"github.com/moby/moby/api/server/router/session"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman run ns", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest PodmanTest
+	)
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanCreate(tempdir)
+		podmanTest.RestoreAllArtifacts()
+		podmanTest.RestoreArtifact(fedoraMinimal)
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+
+	})
+
+	It("podman run pidns test", func() {
+		session := podmanTest.Podman([]string{"run", fedoraMinimal, "bash", "-c", "echo $$"})
```

Comment:
These used to run on Alpine, should we still have a few here?  Or perhaps another set that run on Alpine too?  Just from an overall test perspective, how many tests should we run on alpine, centos, fedora (maybe rhel) containers?  I don't have an answer nor need one, just throwing it up for consumption.

Reply:
the tests werent actually being run, so they never worked ... when I tested them in ginkgo and locally, alpine didn't cut it.

Reply:
also, i worked it to cache fedora-minimal and only unpack it when needed so overall we speed up

Reply:
good to know, thanks for the 411.